src/NuGet.Services.SearchService.Core/Startup.cs

```diff
@@ -88,6 +89,12 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     MaxConnectionsPerServer = 128,
                 });
+
+            services.AddHsts(o =>
+            {
+                o.IncludeSubDomains = true;
+                o.MaxAge = TimeSpan.FromSeconds(31536000);
```

Comment:
Could we use a bigger unit like hours or days to make this a readable number?

```suggestion
                o.MaxAge = TimeSpan.FromDays(365);
```